### PR TITLE
🆕🎨 Design updates 3

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Waterfall PR to implement the new design:
- Refactor shimmer effect and consolidate as a single theme variable
- New footer images (light/dark mode)
- Move 'Polling' from header to footer component + styles
- Style Account Details page
- Appzi button styling
- CoW subsidy modal styling (WIP)
- Network selector fly out menu styling (WIP)
- Menu dropdown fly out menu styling
- Orders panel (activity modal) styling
- Versions (footer) styling

Screen Shot 2022-10-25 at 16 03 17

https://user-images.githubusercontent.com/31534717/197811439-8dc8c99c-787c-4746-8fad-b768d020e9c0.mov

Screen Shot 2022-10-25 at 16 04 20
Screen Shot 2022-10-25 at 16 04 08

https://user-images.githubusercontent.com/31534717/197811447-98c1abbf-98df-4272-a1b6-ed43bb00f8c8.mov

https://user-images.githubusercontent.com/31534717/197811451-17653fc1-0ad6-4f8d-80db-cba300fc0946.mov

### CODE REVIEW

File: src/custom/components/CowSubsidyModal/SubsidyTable.tsx [hunk 1]
```diff
@@ -107,7 +114,7 @@ const SubsidyTr = styled.tr<{ selected?: boolean; darkMode?: boolean }>`
     }
     
     > td {
-      color: ${darkMode ? theme.orange : theme.text1};
+      color: ${darkMode ? theme.text1 : theme.text1};
```
Comment (alfetopito) [inline src/custom/components/CowSubsidyModal/SubsidyTable.tsx:117]: Now both are identical. Unless you intend to change, you can:

```
      color: ${theme.text1};
```

File: src/custom/components/Footer/index.tsx [hunk 2]
```diff
@@ -10,11 +11,14 @@ const Wrapper = styled.div`
   justify-content: space-evenly;
   margin: auto 16px;
   position: relative;
+  width: 100%;
+  margin: 0;
```
Comment (alfetopito) [inline src/custom/components/Footer/index.tsx:15]: Margin is already set on line 12

File: src/custom/components/Header/Polling/index.tsx [hunk 3]
```diff
@@ -1,28 +1,67 @@
...
   }
 
   ${Spinner} {
-    border-left: 2px solid ${({ theme }) => theme.footerColor};
+    border-left: 2px solid ${({ theme }) => theme.text1};
   }
 `
 
-export default function Polling() {
+export const Polling = () => {
```
Comment (alfetopito) [inline src/custom/components/Header/Polling/index.tsx:64]: I'm in favour of no export defaults, but why not keep it a function?
You can remove only the `default` keyword if that's what you want here

File: src/custom/components/OrdersPanel/index.tsx [hunk 4]
```diff
@@ -28,10 +28,9 @@ const SideBar = styled.div`
   cursor: default;
   overflow-y: auto; // fallback for 'overlay'
   overflow-y: overlay;
```
Comment (alfetopito) [inline src/custom/components/OrdersPanel/index.tsx:30]: Not part of this PR, but this property appears twice. Which should we keep?

**Review by alfetopito — COMMENTED** [review]
Couple of comments in the code, will review the behaviour next

File: src/custom/components/CowSubsidyModal/SubsidyTable.tsx (hunk 1 above)
Comment (fairlighteth) [reply src/custom/components/CowSubsidyModal/SubsidyTable.tsx:117]: Yes correct. For now intend to keep for a future change.

File: src/custom/components/Footer/index.tsx (hunk 2 above)
Comment (fairlighteth) [reply src/custom/components/Footer/index.tsx:14]: Thank you, fixed now.

File: src/custom/components/Header/Polling/index.tsx (hunk 3 above)
Comment (fairlighteth) [reply src/custom/components/Header/Polling/index.tsx]: Reverted. I think I saw some TS errors, but turns out those were 'delayed' errors. All good now.

**Comment by alfetopito** [comment]
Enabled lists on light mode are hard to see
Screen Shot 2022-10-26 at 11 14 53

Only thing that I can tell for now.

Otherwise, looks awesome!!

File: src/custom/components/OrdersPanel/index.tsx (hunk 4 above)
Comment (fairlighteth) [reply src/custom/components/OrdersPanel/index.tsx:30]: This is to have a graceful fallback as this 'overlay' property isn't consistently applied: https://caniuse.com/css-overflow-overlay

In particular Safari.

**Review by alfetopito — APPROVED** [review]

**Review by anxolin — APPROVED** [review]
Approved, happy to consolidate this asap so it doesn't conflict to much with develop

As follow up, we can polish things like this text (after ethflow :P ) 
image

Maybe this one:
image

image
